reportService/reportsServer.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable */
-/* This file is deprecated and used as legacy only */
+
```

Comment:
Why did you removed the deprecation comment?

Reply:
b.c. this file is not deprecated. it's part of the puppeteer flow. idk why this comment was there in the first place. maybe @Bars92 can answer